### PR TITLE
Feature/add activeadmin option

### PR DESCRIPTION
**Pull request by Ru-T**
## Why? 
- Users requested option to auto-install ActiveAdmin as the admin interface of choice #34 

## What Changed? 
- Added this as an option, assuming it will be using Devise

### CODE REVIEW

**Comment by dkniffin** [comment]
LGTM, other than the failing tests.

**Comment by Ru-T** [comment]
Thanks @dkniffin. The files generated by ActiveAdmin fail rubocop tests, so fixing those now :)

File: lib/gem_configurator.rb [hunk 1]
```diff
@@ -100,7 +100,7 @@ def smashing_docs?
 def devise_auth?
   if yes?("Add devise_token_auth? (y/n)")
     @devise_auth = true
-    inject_into_file 'Gemfile', after: "gem 'taperole'\n" do
+    inject_into_file 'Gemfile', after: "gem 'pg'\n" do
```
Comment (TylerRockwell) [inline lib/gem_configurator.rb:103]: Temp fix. Looking into a better solution for this.

**Comment by TylerRockwell** [comment]
Addresses #37

**Comment by iandonovan** [comment]
Can this be merged or closed? I'm cleaning out the CR bot.

**Comment by TylerRockwell** [comment]
I'll go ahead and merge it

**Pull request merged**